api/src/test/java/org/openmrs/module/fhir2/providers/r3/ProcedureRequestFhirResourceProviderTest.java

```diff
@@ -107,6 +113,82 @@ public void getServiceRequestByWithWrongId_shouldThrowResourceNotFoundException(
 		assertThat(resourceProvider.getProcedureRequestById(idType), nullValue());
 	}
 	
+	@Test
+	public void createProcedureRequest_shouldCreateNewProcedureRequest() {
+		when(serviceRequestService.create(any(ServiceRequest.class))).thenReturn(serviceRequest);
+		
+		MethodOutcome result = resourceProvider
+		        .createProcedureRequest((ProcedureRequest) VersionConvertor_30_40.convertResource(serviceRequest, false));
+		
+		assertThat(result, notNullValue());
+		assertThat(result.getCreated(), is(true));
+		assertThat(result.getResource(), equalTo(serviceRequest));
```

Comment:
This is definitely not the expected condition (i.e., an R3 resource provider needs to return the R3 resource).